bin/light-base/src/lib.rs

```diff
@@ -261,6 +261,11 @@ struct ChainKey {
 
     /// Network protocol id, found in the chain specification.
     protocol_id: String,
+
+    /// Optional fork identifier, used to differentiate between chains with the same genesis hash.
+    ///
+    /// This can be a counter (e.g. "1", "2", "3") or some unique identifier (e.g. "classic").
+    fork_id: Option<String>,
```

Comment:
@tomaka should `fork_id` be a part of the chain key?

Reply:
Yes!
The `ChainKey` is how we identify that two chains are different.